packages/web/components/earn/allocation/index.tsx

```diff
@@ -101,7 +101,7 @@ export const EarnAllocation = () => {
         </div>
         <small
           className={classNames(
-            "absolute bottom-0 inline-flex w-full justify-center self-center text-overline font-subtitle2 font-medium tracking-normal text-osmoverse-300 transition-opacity duration-200 ease-in-out",
+            "absolute bottom-0 inline-flex w-full justify-center self-center font-subtitle2 font-medium tracking-normal text-osmoverse-300 overline transition-opacity duration-200 ease-in-out",
```

Comment:
This one is off the hook. Multiple fonts are applied. Perhaps looking at earn Figma would reveal the best to use